nav2_bringup/launch/nav2_params.yaml

```diff
@@ -0,0 +1,23 @@
+DwbController:
+  ros__parameters:
+    debug_trajectory_details: True
+    min_vel_x: -0.26
+    min_vel_y: 0.0
+    max_vel_x: 0.26
+    max_vel_y: 0.0
+    max_vel_theta: 1.0
+    min_speed_xy: 0.0
+    max_speed_xy: 0.26
+    min_speed_theta: 0.0
+    acc_lim_x: 2.5
+    acc_lim_y: 0.0
+    acc_lim_theta: 3.2
+    decel_lim_x: -2.5
+    decel_lim_y: 0.0
+    decel_lim_theta: -3.2
+local_costmap:
+  local_costmap:
+    ros__parameters:
+      always_send_full_costmap: True
+      scan:
```

Comment:
I could be missing something here, but is there anywhere in the code looking for the parameter` "scan.topic"`? From the obstacle_layer code, it looks like it is grabbing the `"observation_sources"` topic before  parsing it via stringstream for the`source + "." + "topic" ` parameter.

Reply:
I've removed it for now. We can add obstacle layer params when we are ready to enable the obstacle layer.